### PR TITLE
ROU-4107: Fix RTL issues on layouts

### PR DESCRIPTION
**Pull request by bmarcelino-fe**
This PR is for fixing the issues on reactive and native layouts. Please check the JIRA issue for more information.

### Checklist
-   [X] tested locally
-   [X] documented the code
-   [X] clean all warnings and errors of eslint
-   [ ] requires changes in OutSystems (if so, provide a module with changes)
-   [ ] requires new sample page in OutSystems (if so, provide a module with changes)

### CODE REVIEW

File: src/scss/02-layout/_layout.scss [hunk 1]
```diff
@@ -173,14 +180,26 @@ body,
...
+	}
+
+	.menu-visible {
+		.app-menu-content {
+			right: 0;
+			-webkit-transform: translateX(0) translateZ(0);
+			transform: translateX(0) translateZ(0);
+		}
+	}
+
```
Comment (joselrio) [inline src/scss/02-layout/_layout.scss:195]: These selectors should be placed inside **_menu.scss** partial not inside **_layout.scsc** one.

**Review by joselrio — CHANGES_REQUESTED** [review]
@bmarcelino-fe take a look at my comment related with the selectors inside layout partial instead of being place on menu one.

File: src/scss/02-layout/_layout.scss (hunk 1 above)
Comment (bmarcelino-fe) [reply src/scss/02-layout/_layout.scss:195]: Done

**Review by joselrio — DISMISSED** [review]

File: src/scss/02-layout/_menu-layout-native.scss [hunk 2]
```diff
@@ -29,6 +29,24 @@
 // Responsice --------------------------------------------------------------------
 ///
 .desktop {
+	.layout {
+		&.layout-native {
+			&.layout-side {
+				&.aside-expandable {
+					.app-menu-content {
+						padding-top: calc(var(--header-size) + var(--header-size-content) + var(--os-safe-area-top));
+					}
```
Comment (JoaoFerreira-FrontEnd) [inline src/scss/02-layout/_menu-layout-native.scss:38]: shouldn't the .hide-header-on-scroll:not(.header-is--visible) be here?

File: src/scss/02-layout/_menu-layout-side.scss [hunk 3]
```diff
@@ -108,12 +109,25 @@
...
 		.app-menu-content {
-			left: auto;
 			right: calc(-1 * var(--side-menu-size));
 			transition: all 330ms ease-out;
 		}
 	}
 
+	&:not(.portrait) {
+		.layout {
+			&-side {
```
Comment (JoaoFerreira-FrontEnd) [inline src/scss/02-layout/_menu-layout-side.scss:119]: please concat this line with the 118

**Review by JoaoFerreira-FrontEnd — CHANGES_REQUESTED** [review]

File: src/scss/02-layout/_menu-layout-native.scss (hunk 2 above)
Comment (bmarcelino-fe) [reply src/scss/02-layout/_menu-layout-native.scss:38]: When the layout were created and passed the css to SCSS all of this was tested. At this task, I doesn't change anything on that. I just moved and reorganized the existing rules.

Comment (JoaoFerreira-FrontEnd) [reply src/scss/02-layout/_menu-layout-native.scss:38]: the code is repeated.... is just what I'm saying

File: src/scss/02-layout/_menu-layout-side.scss [hunk 4]
```diff
@@ -108,12 +109,25 @@
...
+				&.layout-native {
+					&.aside-visible {
```
Comment (bmarcelino-fe) [inline src/scss/02-layout/_menu-layout-side.scss:121]: Fixed

File: src/scss/02-layout/_menu-layout-native.scss (hunk 2 above)
Comment (bmarcelino-fe) [reply src/scss/02-layout/_menu-layout-native.scss:38]: Reviewed and fixed. Now are concated and with the right rule

**Review by JoaoFerreira-FrontEnd — APPROVED** [review]

**Review by joselrio — APPROVED** [review]

**Pull request merged**